src/models/user.model.ts

```diff
@@ -1,9 +1,18 @@
-export interface User {
-  login: string;
-  avatar_url: string;
-  public_repos: number;
-  public_gists: number;
-  followers: number;
-  following: number;
-  repos_url: string;
+export class UserEntity   {
+  constructor(
+    public login: string,
+    public name: string,
+    public company: string,
+    public blog: string,
+    public avatar_url: string,
+    public public_repos: number,
+    public public_gists: number,
+    public followers: number,
+    public following: number,
+    public repos_url: string
+  ) {  }
+}
+
+export interface User extends UserEntity{
```

Comment:
Why are you extending an interface from a class? Can't you just add the missing properties in the User and use that? It looks, at least, weird.